File: libraries/hdm/functions/types/request/GetOptionsDTO.js
/* @org.eclipse.vjet.dsf.resource.utils.CodeGen("VjoGenerator") */
vjo.ctype('hdm.functions.types.request.GetOptionsDTO') //< public
.satisfies('java.io.Serializable')
.protos({
    optionName:null, //< private String
    //> public String getOptionName()
    getOptionName:function(){
        return this.optionName;
    },
    //> public void setOptionName(String optionName)
    setOptionName:function(optionName){
        this.optionName=optionName;
    }
})
.endType();

File: libraries/hdm/functions/types/request/GetOptionsDTO.java
package hdm.functions.types.request;

import java.io.Serializable;

public class GetOptionsDTO
  implements Serializable
{
  private String optionName;

  public String getOptionName()
  {
    return this.optionName;
  }

  public void setOptionName(String optionName)
  {
    this.optionName = optionName;
  }
}